guides/advanced-use/asset-compilation.md

```diff
@@ -108,10 +108,10 @@ the `minifyCSS:options` object in your ember-cli-build. Minification is enabled
 default in the production-env and can be disabled using the `minifyCSS:enabled`
 switch.
 
-Similarly, the js-files are minified with `broccoli-uglify-js` in the
-production-env by default. You can pass custom options to the minifier via the
-`minifyJS:options` object in your ember-cli-build. To enable or disable JS minification
-you may supply a boolean value for `minifyJS:enabled`.
+Similarly, the js-files are minified with `ember-cli-uglify`
```

Comment:
`ember-cli-uglify@1.x` has used [`broccoli-uglify-sourcemap`](https://github.com/ember-cli/broccoli-uglify-sourcemap) under the hood. It must be a long, long time ago that [`broccoli-uglify-js`](https://github.com/joliss/broccoli-uglify-js) was used.

`ember-cli-uglify@2.x` uses [`UglifyJS`](https://github.com/mishoo/UglifyJS). But IMO consumer should not care about such implementation details. Therefore I didn't named it.

Reply:
FWIW, we actually use terser now (in more recent versions).